src/app/comments/viewComments/ConfirmationModal.jsx

```diff
@@ -13,7 +13,9 @@ function ConfirmationModal({
 }) {
   return (
     <Modal>
-      <Icon name="exclamation-triangle" size="3x" className="mt-5" />
```

Comment:
Now that `Icon` isn't used anymore, it doesn't need to be imported at the top of the file.

Reply:
Wait, but we use `Icon` for the icons, no? I'm not sure I understand this.

Reply:
Sorry, I meant we don't use `Icon` anymore in _this file_.
I didn't mean we could delete the Icon component - just that importing it in line 3 of this file is now unnecessary.